serde_tests/tests/token.rs

```diff
@@ -429,6 +429,10 @@ impl de::Error for Error {
 
     fn end_of_stream() -> Error { Error::EndOfStreamError }
 
+    fn invalid_value(msg: &str) -> Error {
+        Error::InvalidValue(msg.to_owned())
```

Comment:
Sounds reasonable. I pushed the change. Why do you say we need to get rid of the default impl in 0.8?